src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -61,39 +45,8 @@ const ScheduleActivityPage = ({isLoggedIn, accessToken, userId, activity, links,
       arr.splice(x,1);    // Remember! This deletes an element so the size will decrease by 1.
     }
 
-    return (
-    <div className={classes.root}>
-      <Grid container spacing={3}>
-        {randomArray.map((element) => 
-          <Grid item xs>
-            <Card className={classes.root}>
-
-              <CardContent>
-                <Typography className={classes.title} color="textSecondary" gutterBottom>
-                  {element.category}
-                </Typography>
-                <Typography variant="h6" component="h3">
-                  {element.title}
-                </Typography>
-                <Typography variant="body2" component="p">
-                  {activityType == "active" ? element.creator: activityType == "reading" ? element.description : element.notes}
-                </Typography>
-              </CardContent>
-
-              <CardActions>
-              <div>
-                <Button key={element.key} size="small" variant="contained" color="primary" href={element.url}> VIEW MORE </Button>
-                <Button key={element.key} size="small" color="secondary" variant="contained" onClick={() => { alertUpdateActivity(element) }}> Choose this activity</Button>
-              </div>
-              </CardActions>
-
-            </Card>
-          </Grid>
-        )}
-      </Grid>
-    </div>
-    )
-}
+    return (randomArray);
```

Comment:
Oh, I see what you did here. Good work :)

Reply:
Didn't the ternary operation work? I feel that would make the code more readable and concise. And that's how we've been using across different pages. Thoughts?

Reply:
It doesn't work because we are doing it inside the return function, it reads the <div> as html and thus refuses to see them as arguments.

Reply:
It didn't let me have div tags after '?', it kept giving a type error...

Reply:
Just added the changes.